test/test.js

```diff
@@ -756,6 +756,58 @@ describe('Page', function() {
     }));
   });
 
+  describe('Frame.select', function() {
```

Comment:
... and these tests won't be needed then; the `page.select` tests will cover the whole thing.